Cosa tenure

sakshi1755 opened the pull request:
Created tenure API endpoints. Added create and show tenure feature where u can create cosa tenure records and see them.


![Screenshot 2025-02-13 135546](https://github.com/user-attachments/assets/47aa6d37-9cea-403c-b603-a074a240ab21)

![Screenshot 2025-02-13 135715](https://github.com/user-attachments/assets/81bb6293-674b-472e-ba2a-de4550468fcc)

nishant9083 commented:
You have pushed **node_modules**. While staging files just check what you are adding to commit. Don't do such mistakes.

sakshi1755 commented:
![image](https://github.com/user-attachments/assets/6ccc4e1d-a7ed-4ddb-bbd9-33fef9c86e2d)
![image](https://github.com/user-attachments/assets/2c4dbea1-8af2-4855-b7e0-597b4bcb6acc)
Changed the UI/UX and implemented the suggested changes.

sakshi1755 commented:
![Screenshot (78)](https://github.com/user-attachments/assets/36fa54d2-331e-4edb-b658-2265270dfa7c)
![Screenshot (77)](https://github.com/user-attachments/assets/0ebfb337-eaed-4229-bdeb-f56280b05927)
i used react-toastify library to show errors to the user
